dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -24,7 +24,28 @@
               @change="handleChangeFrameworkEvent"
             />
             <slot name="reportingPeriodDropdown" />
+            <div class="flex align-content-start align-items-center pl-3">
+              <div
+                v-if="
+                  !isReviewableByCurrentUser &&
+                  dataType != DataTypeEnum.EutaxonomyNonFinancials &&
+                  dataType !== DataTypeEnum.Sme
+                "
+                data-test="hideEmptyDataToggle"
+                class="form-field vertical-middle"
+              >
+                <InputSwitch
```

Comment:
I feel like `data-test` for the switch and the parent div are too much and not required for testing. Then I thought that it would probably work to drop the parent div entirely and move the `v-if` and `class` into the switch. I would suppose that this would make no difference in the UI but makes the code slightly compacter/readible.

Reply:
Now that the span with the `Hide empty fields` is there, it would still be visible for all frameworks. Also the ones that do not support the switch. You could move the v-if up to the surrounding div.

Reply:
Should be fixed at last